app/controllers/RateLimitController.scala

```diff
@@ -32,14 +31,20 @@ class RateLimitController @Inject() (
   identify: IdentifierAction,
   getData: DataRetrievalAction,
   val controllerComponents: MessagesControllerComponents,
-  renderer: Renderer
+  renderer: Renderer,
+  requireData: DataRequiredAction,
+  guaranteeBalanceService: GuaranteeBalanceService
 )(implicit ec: ExecutionContext)
     extends FrontendBaseController
     with I18nSupport {
 
   def onPageLoad(): Action[AnyContent] = (identify andThen getData).async {
     implicit request =>
-      val json = Json.obj("nextPageUrl" -> controllers.routes.EoriNumberController.onPageLoad(NormalMode).url)
-      renderer.render("rateLimit.njk", json).map(Ok(_))
+      renderer.render("rateLimit.njk").map(Ok(_))
```

Comment:
Guessing the nextPageUrl isnt required anymore?

Reply:
That's correct.